### PR TITLE
Add more logs for queue operations

### PR DESCRIPTION
**Pull request by ardaguclu**

### CODE REVIEW

**Review by atiratree — COMMENTED** [review]
https://github.com/openshift/cluster-policy-controller/blob/922f9ad6819ff2aa24d17f367bd04a6226d03924/pkg/quota/clusterquotareconciliation/reconciliation_controller.go#L210

is another point of contention that could stop the works. Can you please add to the end of the function?

```go
klog.V(2).Infof("synced cluster resource quota")
```

I have not seen any discovery/monitor failures in KCM in the failed jobs, but it is generally good to know when the syncing has finished since it can freeze the work

File: pkg/quota/clusterquotareconciliation/workqueuebucket.go [hunk 1]
```diff
@@ -91,6 +96,7 @@ func (e *workQueueBucket) GetWithData() (interface{}, []interface{}, bool) {
 		return nil, []interface{}{}, shutdown
 	}
 
+	klog.V(2).Infof("key %s is gotten from queue", key)
```
[the remark this answers is not in the document]
Comment (ardaguclu) [reply pkg/quota/clusterquotareconciliation/workqueuebucket.go:99]: I'll use this one to prove that item is obtained from queue and to ensure that the raciness, I've added `klog.V(2).Infof("lock acquired and key % is added in progress list", key)`

**Comment by ardaguclu** [comment]
Thanks for review @atiratree. Updated according to your suggestions.

File: pkg/quota/clusterquotareconciliation/workqueuebucket.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (atiratree) [reply pkg/quota/clusterquotareconciliation/workqueuebucket.go:99]: ack, let's see what we get with this

**Pull request merged**